tools/scope_upload_classification.py

```diff
@@ -199,8 +199,21 @@ def upload_classification(
                     for t in create_time_list:
                         src_id = create_time_dict[t]
                         if src_id[:4] == 'ZTFJ':
-                            existing_source = src_dict[src_id]
-                            obj_id = src_id
+                            # Treat source as new if not posted to all user-specified groups
+                            posted_groups = [
+                                x['id'] for x in src_dict[src_id]['groups']
+                            ]
+                            n_missing_groups = 0
+                            for gid in group_ids:
+                                if gid not in posted_groups:
+                                    n_missing_groups += 1
+                            if n_missing_groups == 0:
+                                existing_source = src_dict[src_id]
+                                obj_id = src_id
+                            else:
+                                print(
+                                    'Source exists but is not posted to all specified groups. Treating as a new source.'
```

Comment:
I guess this just means posting to the new groups?

Reply:
@mcoughlin Typing my response to your comment led me to make a revision - the code now always has access to the full info stored in the `existing_source` variable (if the source exists). This will provide the downstream code the info it needs to add the source only to groups where it does not already reside, assuming it passes the photometry check.

Reply:
Sounds reasonable.